crmsh/ui_sbd.py

```diff
@@ -102,12 +103,15 @@ class SBD(command.UI):
     PCMK_ATTRS_DISKLESS = ('stonith-watchdog-timeout',)
     PARSE_RE = re.compile(
 		# Match keys with non-empty values, capturing possible suffix
-        r'(\w+)(?:-(\w+))?=("[^"]+"|[\w/\d;]+)'
+        r'([\w-]+)-([\w-]+)=([\w/\d]+)'
```

Comment:
Comment and code disagree here.

Reply:
Update as
```Python
    PARSE_RE = re.compile(
        # To extract key, suffix and value from these possible arguments:
        # watchdog-timeout=30
        # crashdump-watchdog-timeout=120
        # watchdog-device=/dev/watchdog
        r'([\w-]+)-([\w]+)=([\w/]+)'
    )
```
And added the unit test for it